### ISSUE TITLE
Raise an error if object isn't valid

### ISSUE THREAD

**Issue by FanaHOVA** [davydovanton/shallow_attributes#4]
If an object is created that doesn't satisfy the validations there's no errors raised, but you can see them in `object.errors`. An error should be raised to warn the user. Example:

```
class User
  include ShallowAttributes

  attribute :name, String
  validates :name, presence: true
end
```

```
> u = User.new(name: nil)
> #<TextDispatcher name=nil>
> u.errors
>  #<ActiveModel::Errors:0x007fc153323090 @base=#<TextDispatcher name=nil>, @messages={:name=>["can't be blank"]}>
```

I'd expect

```
> u = User.new(name: nil)
> Error: name can't be blank
```

**Comment by nesaulov** [comment]
I guess this can be closed now (due to #17)

**Comment by davydovanton** [comment]
@nesaulov yep, thanks!
